demo/prometheus/prometheus.yml

```diff
@@ -0,0 +1,15 @@
+global:
+  scrape_interval: 15s
+  evaluation_interval: 15s
+
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs:
+      - targets:
+        - '127.0.0.1:9090'
+
+  - job_name: 'layotto'
+    scheme: http
+    static_configs:
+      - targets:
+        - 'docker.for.mac.localhost:34903'
```

Comment:
Does it mean that this quickstart can only be run under MacOS?
If it is, we can add a statement at the beginning of the quickstart document saying that "This quickstart can only be run under MacOS. "